addons/account/models/account_payment.py

```diff
@@ -896,12 +896,13 @@ def create(self, vals_list):
                     pay.move_id.write(move_vals)
         return payments
 
-    def _get_outstanding_account(self, payment_type):
+    def _get_outstanding_account(self, payment_type, company_id=None):
```

Comment:
@william-andre I added the possibility to pass a company, so that the fallback on `transfer_account_id` would work in my case. If this doesn't seem right to you we can maybe go back to the previous solution and directly fallback on `transfer_account_id`

Reply:
This is fine :+1: